### PR TITLE
Fix images on prod

### PR DESCRIPTION
**Pull request by vidoseaver**
Add  .erb  to  application.css
Change background image in css file to use asset tag and ERB
Add erb extention to about_body.js.JSX
Change img src tags to erb helpers.

### CODE REVIEW

**Comment by neight-allen** [comment]
This seems to have more than just the images fix in it. Could you isolate your changes to a branch from the current develop?

**Comment by vidoseaver** [comment]
I kind of did, because development was broken and I was waiting for a merge. This was the closest working branch.  Now I need it merged to dev so that I can pull down dev, update staging, remove the staging remote from heroku and set it to auto update on that.

**Comment by bfpepper** [comment]
Looks good. Ready to merge when everyone else comments.

File: app/assets/javascripts/components/_about_body.js.jsx.erb [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 var AboutBody = React.createClass({
   render() {
     return (
       <div className="container ">
         <div id="about-container">
           <div className="row">
             <div className="col s4 about_anchor">
-              <img src={"assets/about-cartoon-anchor.png"} />
+              <%= image_tag 'about-cartoon-anchor' %>
+
```
Comment (meganft) [inline app/assets/javascripts/components/_about_body.js.jsx.erb]: rails helpers!

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -5,7 +5,6 @@ def create
     user = User.find_or_create_by(id: census_user_info["uid"])
     user.token = census_user_info['info']['token']
     user.census_id = census_user_info['uid']
-    redirect_to dashboard_path(user)
+    redirect_to edit_dashboard_path(user)
```
Comment (meganft) [inline app/controllers/sessions_controller.rb]: 👍

**Comment by jbkimble** [comment]
👍 Looks good!

**Pull request merged**